### PR TITLE
86cupmdug/error page

### PR DESCRIPTION
**Pull request by sania-salim**
This page contains the error page that will be shown when a user provides a route that does not exist.

**changes made:**

- Error message and illustration
- Buttons and navigation
- Language support for error page

### CODE REVIEW

File: src/pages/ErrorPage/ErrorPage.styled.ts [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+    justify-content: center;
+    align-items: center;
+`;
+
+export const ErrorHeading = styled.h1`
+    padding: 5rem;
+    font-size: 3rem;
+`;
+
+export const ButtonContainer = styled.div`
```
Comment (RJR-003) [inline src/pages/ErrorPage/ErrorPage.styled.ts]: Please use MUI components for uniformity.

**Review by RJR-003 — COMMENTED** [review]
Please give some top margin to the entire page.
Please adjust the spacing between image and other elements.

File: src/pages/ErrorPage/ErrorPage.tsx [hunk 2]
```diff
@@ -1,9 +1,50 @@
...
+    };
+
+    const goBackHandler = () => {
+        navigate(-1);
+    };
+
+    return (
+        <ErrorPageWrapper>
+            <ErrorHeading>{t('errorPageHeading')}</ErrorHeading>
+            <img src={lost} />
```
Comment (abhijith-biju) [inline src/pages/ErrorPage/ErrorPage.tsx]: Would be good to reduce size of image for smaller screens.

File: src/pages/ErrorPage/ErrorPage.tsx [hunk 3]
```diff
@@ -1,9 +1,50 @@
...
-  return (
-    <ErrorPageWrapper>
-      <h1>404 Page Not Found</h1>
-    </ErrorPageWrapper>
-  );
+    const navigate = useNavigate();
+    const { t } = useTranslation('errorPage');
+
+    const goHomeHandler = () => {
+        navigate('/');
```
Comment (abhijith-biju) [inline src/pages/ErrorPage/ErrorPage.tsx]: Would be better to use paths.home variable.

**Review by abhijith-biju — COMMENTED** [review]
Please update changelog.

File: src/pages/ErrorPage/ErrorPage.tsx (hunk 2 above)
Comment (sania-salim) [reply src/pages/ErrorPage/ErrorPage.tsx]: Have used useMediaquery to resize the image according to screen size in commit https://github.com/nijin-26/bus-ticket-booking-react/pull/28/commits/f63a90398901d22dd4d6a3b20da04defd7cb8acf

**Review by RJR-003 — APPROVED** [review]

**Review by abhijith-biju — APPROVED** [review]

**Review by lekshmi615213 — APPROVED** [review]

**Pull request merged**